rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/LRATest.java

```diff
@@ -9,12 +9,7 @@
 import static org.eclipse.microprofile.lra.annotation.ws.rs.LRA.LRA_HTTP_CONTEXT_HEADER;
 import static org.eclipse.microprofile.lra.annotation.ws.rs.LRA.LRA_HTTP_RECOVERY_HEADER;
 import static org.hamcrest.Matchers.containsString;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
I would have preferred to see the individual imports remain rather than the .* but I won't request changes on the pull request for that change to this test.

Reply:
Thank you!

Reply:
Actually, I'll just squash the top two commits.